app-editors/helix/helix-23.05.ebuild

```
@@ -0,0 +1,322 @@
+	idna-0.3.0
+	ignore-0.4.20
+	imara-diff-0.1.5
+	indexmap-1.9.3
+	indoc-2.0.1
+	instant-0.1.12
+	io-close-0.3.7
+	io-lifetimes-1.0.10
+	itoa-1.0.6
+	js-sys-0.3.61
+	kstring-2.0.0
+	lazy_static-1.4.0
+	libc-0.2.144
+	libloading-0.8.0
+	link-cplusplus-1.0.8
+	linux-raw-sys-0.3.4
+	lock_api-0.4.9
+	log-0.4.17
+	lsp-types-0.94.0
+	memchr-2.5.0
+	memmap2-0.5.10
+	minimal-lexical-0.2.1
+	miniz_oxide-0.6.2
+	mio-0.8.6
+	nom-7.1.3
+	num-integer-0.1.45
+	num-traits-0.2.15
+	num_cpus-1.15.0
+	num_threads-0.1.6
+	once_cell-1.17.1
+	parking_lot-0.12.1
+	parking_lot_core-0.9.7
+	percent-encoding-2.2.0
+	pin-project-lite-0.2.9
+	pin-utils-0.1.0
+	proc-macro2-1.0.56
+	prodash-23.1.2
+	pulldown-cmark-0.9.2
+	quickcheck-1.0.3
+	quote-1.0.26
+	rand-0.8.5
+	rand_core-0.6.4
+	redox_syscall-0.2.16
+	redox_syscall-0.3.5
+	regex-1.8.1
+	regex-automata-0.1.10
+	regex-syntax-0.6.29
+	regex-syntax-0.7.1
+	ropey-1.6.0
+	rustix-0.37.15
+	ryu-1.0.13
+	same-file-1.0.6
+	scopeguard-1.1.0
+	scratch-1.0.5
+	serde-1.0.163
+	serde_derive-1.0.163
+	serde_json-1.0.96
+	serde_repr-0.1.12
+	serde_spanned-0.6.1
+	sha1_smol-1.0.0
+	signal-hook-0.3.15
+	signal-hook-mio-0.2.3
+	signal-hook-registry-1.4.1
+	signal-hook-tokio-0.3.1
+	slab-0.4.8
+	slotmap-1.0.6
+	smallvec-1.10.0
+	smartstring-1.0.1
+	smawk-0.3.1
+	socket2-0.4.9
+	static_assertions-1.1.0
+	str-buf-1.0.6
+	str_indices-0.4.1
+	syn-1.0.109
+	syn-2.0.15
+	tempfile-3.5.0
+	termcolor-1.2.0
+	termini-1.0.0
+	textwrap-0.16.0
+	thiserror-1.0.40
+	thiserror-impl-1.0.40
+	thread_local-1.1.7
+	threadpool-1.8.1
+	time-0.3.20
+	time-core-0.1.0
+	time-macros-0.2.8
+	tinyvec-1.6.0
+	tinyvec_macros-0.1.1
+	tokio-1.28.1
+	tokio-macros-2.1.0
+	tokio-stream-0.1.14
+	toml-0.7.3
+	toml_datetime-0.6.1
+	toml_edit-0.19.8
+	tree-sitter-0.20.10
+	unicase-2.6.0
+	unicode-bidi-0.3.13
+	unicode-bom-2.0.2
+	unicode-general-category-0.6.0
+	unicode-ident-1.0.8
+	unicode-linebreak-0.1.4
+	unicode-normalization-0.1.22
+	unicode-segmentation-1.10.1
+	unicode-width-0.1.10
+	url-2.3.1
+	version_check-0.9.4
+	walkdir-2.3.3
+	wasi-0.11.0+wasi-snapshot-preview1
+	wasm-bindgen-0.2.84
+	wasm-bindgen-backend-0.2.84
+	wasm-bindgen-macro-0.2.84
+	wasm-bindgen-macro-support-0.2.84
+	wasm-bindgen-shared-0.2.84
+	which-4.4.0
+	winapi-0.3.9
+	winapi-i686-pc-windows-gnu-0.4.0
+	winapi-util-0.1.5
+	winapi-x86_64-pc-windows-gnu-0.4.0
+	windows-0.48.0
+	windows-sys-0.45.0
+	windows-sys-0.48.0
+	windows-targets-0.42.2
+	windows-targets-0.48.0
+	windows_aarch64_gnullvm-0.42.2
+	windows_aarch64_gnullvm-0.48.0
+	windows_aarch64_msvc-0.42.2
+	windows_aarch64_msvc-0.48.0
+	windows_i686_gnu-0.42.2
+	windows_i686_gnu-0.48.0
+	windows_i686_msvc-0.42.2
+	windows_i686_msvc-0.48.0
+	windows_x86_64_gnu-0.42.2
+	windows_x86_64_gnu-0.48.0
+	windows_x86_64_gnullvm-0.42.2
+	windows_x86_64_gnullvm-0.48.0
+	windows_x86_64_msvc-0.42.2
+	windows_x86_64_msvc-0.48.0
+	winnow-0.4.1
+"
+
+inherit bash-completion-r1 cargo desktop xdg-utils
+
+DESCRIPTION="A post-modern text editor."
```

```suggestion
DESCRIPTION="A post-modern text editor"
```
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text